### ISSUE TITLE
Add support to allow ports as variables in docker compose 

### ISSUE THREAD

**Issue by tushar5526** [tushar5526/sarthi#46]

**Comment by Amandeepsingh25** [comment]
I'm eager to contribute and resolve the Docker Compose port variable issue.I would like to contribute to saarthi.Please assign me this issue @tushar5526

**Comment by GJS2162** [comment]
Hi Tushar,

I would like to take on this task of adding support to allow ports as variables in the Docker Compose file (#46). I have experience working with Docker and Docker Compose, and I believe I can implement this feature efficiently.

Please assign this task to me.

Thank you!

Best regards,
Gaurav Ji Srivastava

**Comment by Amandeepsingh25** [comment]
#53

**Comment by Amandeepsingh25** [comment]
**Enhancement: Flexible Port Configuration in Docker Compose#46   #53**

**Comment by Amandeepsingh25** [comment]
#53

**Comment by tushar5526** [comment]
Hey @Amandeepsingh25 @GJS2162 great first try but I think there was a miss on my end to explain the problem statements in a better way. 

First of all I wanted to know were your folks able to setup the project locally ?

**Comment by Amandeepsingh25** [comment]
@tushar5526  yes I have run the docker compose file locally but fail to unerstand the endpoint for knowing the output .

**Comment by tushar5526** [comment]
@Amandeepsingh25 check out the contributor's guide in the readme.

**Comment by GJS2162** [comment]
> Hey @Amandeepsingh25 @GJS2162 great first try but I think there was a miss on my end to explain the problem statements in a better way.
> 
> First of all I wanted to know were your folks able to setup the project locally ?

Sorry for the late reply.

Initially, I reviewed the documentation to understand how to run the project. However, I decided to proceed directly with the task at hand. I will give it a try, and if I encounter any issues or need assistance, I will reach out for help.

Thank you!
